pkgs/applications/system/zxfer/default.nix

```
@@ -0,0 +1,81 @@
+{ lib
+, bash
+, fetchFromGitHub
+, installShellFiles
+, makeWrapper
+, coreutils
+, gawk
+, gnugrep
+, gnused
+, openssh
+, resholve
+, rsync
+, zfs
+}:
+
+resholve.mkDerivation rec {
+  pname = "zxfer";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    owner = "allanjude";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-11SQJcD3GqPYBIgaycyKkc62/diVKPuuj2Or97j+NZY=";
+  };
+
+  buildPhase = ''
+    substituteInPlace zxfer \
+      --replace 'AWK=$( which awk )'     'AWK=${gawk}/bin/awk' \
+      --replace 'AWK=$( which gawk )'    'AWK=${gawk}/bin/awk' \
+      --replace 'LCAT=""'                'LCAT=${coreutils}/bin/cat' \
+      --replace 'LZFS=$( which zfs )'    'LZFS=${zfs}/bin/zfs' \
+      --replace 'RSYNC=$( which rsync )' 'RSYNC=${rsync}/bin/rsync'
+  '';
+
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    installManPage zxfer.1m zxfer.8
+    install -Dm755 zxfer -t $out/bin/
+
+    runHook postInstall
+  '';
+
+  solutions.default = {
+    scripts = [ "bin/zxfer" ];
+    interpreter = "${bash}/bin/bash";
```

```suggestion
    interpreter = "${bash}/bin/sh";
```

This one also uses a /bin/sh shebang https://github.com/allanjude/zxfer/blob/master/zxfer#L1